code/src/components/Age.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+
+const ageGroups = ['0-19','20-29','30+'] 
+
+export const AgeRadioOption = ({ageGroupSpan, setAgeGroup}) => {
+  
+  return (
+  <div className='age-input'>
+    <p>Age</p>
+      {ageGroups.map((age) => (
+        <label htmlFor='age' key={age}>
+        <input
+        name='age'
+        type="radio"
+        value={age}
+        onChange={(event) => setAgeGroup(event.target.value)}
```

Comment:
I would consider this function that you’ve put here directly on the onChange attribute a bit. This code is working and this is one way to do this. But in the spirit of consistency I suggest that you stick with one method on how to handle the event when the user is putting in a value here (by choosing a radio button) which you wrote differently in the other components (you made a separate function). 🧐